StitchCoreServicesMongoDbRemote/Sources/StitchCoreServicesMongoDbRemote/RemoteInsertManyResult.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import MongoSwift
+
+/// The result of an `insertMany` command on a `RemoteMongoCollection`.
+public struct RemoteInsertManyResult: Decodable {
+    /// Map of the index of the inserted document to the id of the inserted document.
+    public let insertedIds: [Int64: BsonValue]
```

Comment:
[minor] This doesn't seem to be used. What is this for?

Reply:
An insert many returns the ids that have been inserted.

Reply:
Why are we mapping it with its indexes? It seems to return an Array of insertedIds. That was more the question I was getting at.

Reply:
The crud spec (https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst) alludes to it but doesn't go into detail. An insert many can be treated as unordered or ordered. In the former case a sparse part of the inserted documented may be inserted so this allows for a sparse mapping of inserted ids relative to their order.

Reply:
Yeah I went with this because it was the way the Swift driver exposed it